docs/src/css/custom.scss

```diff
@@ -22,6 +22,9 @@
 
 	// Workaround for <https://github.com/easyops-cn/docusaurus-search-local/issues/336>
 	--ifm-navbar-search-input-icon: url("@site/static/assets/search-icon-light.svg");
+
+	//Text color for classes in code blocks
+	--ifm-attr-name-color: #345094;
```

Comment:
For these overrides, would you mind linking to the documentation for the CSS attributes we're changing?